### DISCUSSION TITLE
Add lint checks on CI

### DISCUSSION THREAD

**Discussion by subzero10** [honeybadger-io/honeybadger-js#738] [Ideas]
I have noticed that for some projects we have eslint enabled but it is only executed manually. For some projects, we don't have eslint configured at all.

Personally, I use my IDE to automatically lint on file save when eslint is configured on a project. However, this is not a guaranteed solution for all contributors.

I think we should add lint checks on CI for all projects. Or even a git hook to lint before commit. Or both. 

What do you think?

**Comment by joshuap** [comment]
Both works for me. 👍 

Let's automate all the things. :)

**Reply by joshuap** [reply]
I also want to make sure we use the same javascript style across all projects.

**Reply by subzero10** [reply]
Awesome, I created an issue for the js repo for now.

**Reply by subzero10** [reply]
> I also want to make sure we use the same javascript style across all projects.

Same, that's why I want to add the check on CI.

**Reply by shalvah** [reply]
Ah, I'm personally not a fan of restrictive code style checks. But as long as it stays out of the way (not too many rules, not overly nitpicking), I can live with it.

**Reply by subzero10** [reply]
I agree. We'll probably go with a standard one and disable a few. I guess we can discuss that on the PR. Plus, we can always revise them.
